src/06-serial-communication/nix-tooling.md

```diff
@@ -19,42 +19,9 @@ the board; check the next section, which contains instructions for older
 revisions. If you do have a newer revision skip the next section and move to the
 "minicom" section.
 
-## Older revisions of the discovery board / external serial module
+## minicom
 
-Connect the serial module to your computer and let's find out what name the OS assigned to it.
-
-> **NOTE** On macs, the USB device will named like this: `/dev/cu.usbserial-*`. You won't
-> find it using `dmesg`, instead use `ls -l /dev | grep cu.usb` and adjust the following 
-> commands accordingly!
-
-``` console
-$ dmesg | grep -i tty
-(..)
-[  +0.000155] usb 3-2: FTDI USB Serial Device converter now attached to ttyUSB0
-```
-
-But what's this `ttyUSB0` thing? It's a file of course! Everything is a file in \*nix:
-
-``` console
-$ ls -l /dev/ttyUSB0
-crw-rw-rw- 1 root uucp 188, 0 Oct 27 00:00 /dev/ttyUSB0
-```
-
-> **NOTE** if the permissions above is `crw-rw----`, the udev rules have not been set correctly
-> see [udev rules](../03-setup/linux.html#udev-rules)
-
-You can send out data by simply writing to this file:
-
-``` console
-$ echo 'Hello, world!' > /dev/ttyUSB0
-```
-
-You should see the TX (red) LED on the serial module blink, just once and very fast!
```

Comment:
Except for the different device name, this can still be done, right?

Reply:
True, I'll add it back in!